### PR TITLE
Link postgresql

### PR DESCRIPTION
**Pull request by posix4e**

### CODE REVIEW

**Comment by posix4e** [comment]
What should we do to merge this

**Comment by solidsnack** [comment]
Most of the settings in this branch in the `build.rs` files are not needed, as per #10, which builds on both Mac and Linux, with a ton of them removed. Those changes should find their way into this branch.

File: elephantpump-sys/src/build.rs [hunk 1]
```diff
@@ -0,0 +1,18 @@
+extern crate pkg_config;
+
+fn main ()
+{
+    match pkg_config::find_library("pgcommon") {
+        Ok(_) => {
+            if cfg!(target_os = "macos") {
+                println!("cargo:rustc-flags=-L /usr/local/lib -lpgcommon");
```
Comment (solidsnack) [inline elephantpump-sys/src/build.rs:8]: These can probably be removed without affecting anything.

File: src/build.rs [hunk 2]
```diff
@@ -0,0 +1,10 @@
+fn main() {
+    println!("cargo:rustc-link-search=native=/usr/local/opt/openssl/lib");
```
Comment (solidsnack) [inline src/build.rs:2]: As long as the linking is happening in the `-sys` module then we don't need any linker flags at this level.

**Pull request closed without merging**